modules/standard/IO.chpl

```diff
@@ -1516,50 +1546,68 @@ extern record iostyle { // aka qio_style_t
   /* When performing text I/O, do not use more than this many bytes */
   var max_width_bytes:uint(32) = max(uint(32));
 
-  /* What character do we start strings with, when appropriate? */
+  /* What character do we start strings with, when appropriate? Default is " */
   var string_start:style_char_t = 0x22; // "
```

Comment:
Future todo: It could be nice to use something like the `ascii` function with a param string, to avoid having these comments.